test/markup/xml/document-type-variations.expect.txt

```diff
@@ -0,0 +1,38 @@
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> svg</span><span class="hljs-meta-keyword"> PUBLIC</span> <span class="hljs-meta-string">" -//W3C//DTD SVG 1.1 Basic//EN "</span> <span class="hljs-meta-string">' http://www.w3.org/Graphics/SVG/1.1/DTD/svg11-basic.dtd '</span> &gt;</span>
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> svg</span><span class="hljs-meta-keyword"> PUBLIC</span><span class="hljs-meta-string">"-//W3C//DTD SVG 1.1 Basic//EN"</span><span class="hljs-meta-string">'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11-basic.dtd'</span>&gt;</span>
+
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> note</span> [
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> nbsp</span> <span class="hljs-meta-string">'&amp;#xA0;'</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> pound</span> <span class="hljs-meta-string">'&amp;#xA3;'</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> writer_person</span> <span class="hljs-meta-string">" Writer: Donald Duck. "</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> copyright</span> <span class="hljs-meta-string">"Copyright: Walt Disney Company."</span>&gt;</span>
+]&gt;</span>
+
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> nbsp</span> <span class="hljs-meta-string">' &amp;#xA0; '</span>&gt;</span>
```

Comment:
The space should not be a part of the keyword.

Reply:
I wanted to save a step, but I am changing it so that the spaces are not in the markup.